src/serde_utils.rs

```diff
@@ -147,23 +151,22 @@ pub mod btreemap_as_seq_byte_values {
 
     // NOTE: This module can be exactly copied to use with HashMap.
 
-    use crate::prelude::*;
     use serde;
 
+    use crate::prelude::*;
+
     /// A custom key-value pair type that serialized the bytes as hex.
     #[derive(Debug, Deserialize)]
     struct OwnedPair<T>(
         T,
-        #[serde(deserialize_with = "crate::serde_utils::hex_bytes::deserialize")]
-        Vec<u8>,
+        #[serde(deserialize_with = "crate::serde_utils::hex_bytes::deserialize")] Vec<u8>,
```

Comment:
I can't work out how to stop this happening?

Reply:
It does read like this should control it but its currently set to 0 (default) so should not be happening?